e2e/_suites/kubernetes-autodiscover/features/filebeat.feature

```diff
@@ -1,38 +1,35 @@
-@kubernetes
-@autodiscover
+@kubernetes-autodiscover
 @filebeat
-Feature: Log collection from Kubernetes pods.
+Feature: Filebeat
   Use Kubernetes autodiscover features in Filebeat to collect logs.
 
-Scenario: Logs collection from running pod
+Background:
```

Comment:
About the background, I prefer having scenarios being self-content, with no need to check background clauses at the top of the file, although in this particular case the feature file is small.

In the case the feature file grows, we can skip the Background lint rule, but let's keep it as is by the moment

Reply:
Same thing, I would also prefer not to use `Background`, but the linter asks for it:
```
UseBackground - Step 'Given "filebeat" is running with "hints enabled"' should be part of background
  e2e/_suites/kubernetes-autodiscover/features/filebeat.feature (4): Log collection from Kubernetes pods.
```